pdns/pdns_recursor.cc

```diff
@@ -1401,6 +1401,18 @@ static void startDoResolve(void *p)
     }
   sendit:;
 
+    if(g_useIncomingECS && haveEDNS && !sr.wasVariable()) {
```

Comment:
We might want to also check that `variableAnswer` was not set, because that's the only way a `Lua` script has to tell us that it took a decision based on the source.